code/sorting/topological_sort/topological_sort.cpp

```diff
@@ -1,97 +1,55 @@
-// A C++ program to print topological sorting of a DAG
+#include <bits/stdc++.h>
```

Comment:
We prefer to don't use `<bits/stdc++.h>` here.
list all headers what this program actually needs is better.